public/javascript/ep-card-filter.js

```diff
@@ -194,9 +199,45 @@ $(document).ready(function(){
       window.cards.saveState();
     });
 
-    $('[data-section-toggle]').on('click', function(){
-      var section = $(this).attr('data-section-toggle');
+    $('.js-show-facet').on('change', function(){
```

Comment:
> we can change to the shorthand methods in a future PR

Interesting. My vote would always be for using `.on()` everywhere – although I'd be interested to hear the argument for shorthand methods, aside from being five characters shorter.